query/src/main/scala/filodb/query/exec/MetadataExecPlan.scala

```diff
@@ -46,7 +46,7 @@ trait MetadataDistConcatExec extends NonLeafExecPlan {
       case (QueryError(_, _, ex), _)         => throw ex
     }.toListL.map { resp =>
       val metadataResult = scala.collection.mutable.Set.empty[Map[ZeroCopyUTF8String, ZeroCopyUTF8String]]
-      resp.foreach { rv =>
+      resp.filter(!_.isEmpty).foreach { rv =>
```

Comment:
Not needed here. Everything eventually flows through ExecPlan.scala, so it will eventually be filtered.

Reply:
Some metadata tests fail if I undo this-- `rv` is a (possibly empty) sequence of `RangeVector`s.

Reply:
@sherali42  Can you help confirm if removing RV when no rows will not cause regression for metadata queries ?

Reply:
nope, we should be good